### ISSUE TITLE
typescript support?

### ISSUE THREAD

**Issue by HeisenbergRoc** [megafetis/vue3-treeselect#7]
can provide @types/ declare module support typescript

**Comment by tmarcis1** [comment]
Yes, this'd be extremely helpful!  Many thanks for the full-featured component and the Vue3 implementation.
